F1WM/Mapping/NewsMappingProfile.cs

```diff
@@ -10,8 +10,9 @@ public class NewsMappingProfile : Profile
 		public NewsMappingProfile()
 		{
 			CreateMap<News, NewsSummary>()
-				.ForMember(api => api.TopicIcon, o => o.MapFrom(db => db.Topic.TopicIcon));
-			CreateMap<News, NewsDetails>();
+				.ForMember(api => api.TopicIcon, o => o.MapFrom(db => db.Topic.TopicIcon))
+                .ForMember(api => api.IsHighlighted, o => o.MapFrom(db => db.NewsHighlight));
```

Comment:
Actually, we could save this line if both properties were named the same. It'd work either way, but probably IsHighlighted is a more accurate name, so I'd opt for changing property name in the database model.